src/test/java/com/redislabs/redisgraph/RedisGraphAPITest.java

```diff
@@ -12,247 +13,77 @@ public RedisGraphAPITest() {
         api = new RedisGraphAPI("social");
     }
 
-    @org.testng.annotations.BeforeSuite
-    @org.testng.annotations.BeforeMethod
+    @Before
     public void flushDB() {
-        api.deleteGraph();
+    	api.deleteGraph();
     }
 
-    @org.testng.annotations.Test
+    
+    @Test
     public void testCreateNode() throws Exception {
-        // Create a node
-        RedisNode node = api.createNode("name", "roi", "age", 32);
-
-        // Validate node
-        Assert.assertNotNull(node.getId());
-        Assert.assertSame(node.getAttributes().get("name"), "roi");
-        Assert.assertSame(Integer.parseInt(node.getAttributes().get("age")), 32);
-        Assert.assertSame(node.getAttributes().size(),2);
+        // Create a node    	
+    	ResultSet result = api.query("CREATE ({name:\"roi\",age:32})");
+    	Assert.assertFalse(result.hasNext());
+    	
+    	Assert.assertEquals("1", result.getStatistics().getStringValue(Label.NODES_CREATED));
```

Comment:
Can we do result.getStatistics().nodesCreated() ?

Reply:
@swilly22 Added accessors